packages/ui/src/styles/globals.css

```
@@ -1,10 +1,11 @@
 @import "tailwindcss";
 @source "../components/**/*.{ts,tsx}";
 
+@import "./zIndex.css";
```

Stylelint (17.3.0)

[error] Unexpected invalid position `@import` rule (no-invalid-position-at-import-rule)

(no-invalid-position-at-import-rule)